ts/src/bybit.ts

```diff
@@ -3927,11 +3927,11 @@ export default class bybit extends Exchange {
         } else if (timeInForce === 'ioc') {
             request['timeInForce'] = 'IOC';
         }
-        let triggerPrice = this.safeNumber2 (params, 'triggerPrice', 'stopPrice');
-        const stopLossTriggerPrice = this.safeNumber (params, 'stopLossPrice');
-        const takeProfitTriggerPrice = this.safeNumber (params, 'takeProfitPrice');
-        const stopLoss = this.safeNumber (params, 'stopLoss');
-        const takeProfit = this.safeNumber (params, 'takeProfit');
+        let triggerPrice = this.safeValue2 (params, 'triggerPrice', 'stopPrice');
```

Comment:
@Dan-krm I understand this is the unified format but now would be a breaking-change, can we maybe support both?

Reply:
Yeah I adjusted stopLoss and takeProfit so that this still works:
```
bybit createOrder BTC/USDT:USDT limit buy 0.001 23000 '{"stopLoss":"22000","positionIdx":1}'
```
Let me know if there is anything else